src/cf-forms/src/atoms/select.less

```diff
@@ -0,0 +1,68 @@
+.a-select {
+    position: relative;
+    border: 1px solid @select-border;
+
+    select {
+        height: unit( 30px / @base-font-size-px, em );
+        width: 100%;
+        padding: unit( 4px / @base-font-size-px, em )
+                 unit( 6px / @base-font-size-px, em );
+        border: 0;
+        outline: 2px solid transparent;
+        appearance: none;
+        background-color: @input-bg;
+        border-radius: 0;
+        color: @text;
+
+        &:hover,
+        &:active,
+        &:focus {
+            outline-color: @input-border__focused;
+            outline-offset: 0;
```

Comment:
This happens without the rule. 
![image](https://cloud.githubusercontent.com/assets/1860176/18766242/45374692-80e7-11e6-950a-a91512636ff5.png)